sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -289,7 +289,7 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 				}
 			}
 
-			if env.LocalImageScanningEnabled.BooleanSetting() {
+			if env.LocalImageScanningEnabled.BooleanSetting() && !env.DelegatedScanningDisabled.BooleanSetting() {
```

Comment:
can you add a comment above this to make it clear anything within this closure is related to delegated scanning?